README.md

```diff
@@ -1 +1,23 @@
-# QueueToken.V1.Ruby
\ No newline at end of file
+# QueueToken.V1.Ruby
+
```

Comment:
Let's add a few words of descriptions!

Reply:
I adapted the README of the Java version 😃